addons/knobs/src/components/__tests__/Panel.js

```diff
@@ -23,120 +23,60 @@ describe('Panel', () => {
-        on: (e, handler) => {
+      const testChannel = { on: (e, handler) => {
           handlers[e] = handler;
-        },
-        emit: jest.fn(),
-      };
+        }, emit: jest.fn() };
 
-      const testQueryParams = {
-        'knob-foo': 'test string',
-        bar: 'some other string',
-      };
+      const testQueryParams = { 'knob-foo': 'test string', bar: 'some other string' };
 
-      const testApi = {
-        getQueryParam: key => testQueryParams[key],
-        setQueryParams: jest.fn(),
-        onStory: jest.fn(),
-      };
+      const testApi = { getQueryParam: key => testQueryParams[key], setQueryParams: jest.fn(), onStory: jest.fn() };
 
       shallow(<Panel channel={testChannel} api={testApi} />);
       const setKnobsHandler = handlers['addon:knobs:setKnobs'];
 
-      const knobs = {
-        foo: {
-          name: 'foo',
-          value: 'default string',
-          type: 'text',
-        },
-        baz: {
-          name: 'baz',
-          value: 'another knob value',
-          type: 'text',
-        },
-      };
+      const knobs = { foo: { name: 'foo', value: 'default string', type: 'text' }, baz: { name: 'baz', value: 'another knob value', type: 'text' } };
 
       setKnobsHandler({ knobs, timestamp: +new Date() });
-      const knobFromUrl = {
-        name: 'foo',
-        value: testQueryParams['knob-foo'],
-        type: 'text',
-      };
+      const knobFromUrl = { name: 'foo', value: testQueryParams['knob-foo'], type: 'text' };
       const e = 'addon:knobs:knobChange';
       expect(testChannel.emit).toHaveBeenCalledWith(e, knobFromUrl);
     });
 
     it('should set query params when url params are already read', () => {
       const handlers = {};
 
-      const testChannel = {
-        on: (e, handler) => {
+      const testChannel = { on: (e, handler) => {
           handlers[e] = handler;
-        },
-        emit: jest.fn(),
-      };
+        }, emit: jest.fn() };
 
-      const testQueryParams = {
-        'knob-foo': 'test string',
-        bar: 'some other string',
-      };
+      const testQueryParams = { 'knob-foo': 'test string', bar: 'some other string' };
 
-      const testApi = {
-        getQueryParam: key => testQueryParams[key],
-        setQueryParams: jest.fn(),
-        onStory: jest.fn(),
-      };
+      const testApi = { getQueryParam: key => testQueryParams[key], setQueryParams: jest.fn(), onStory: jest.fn() };
 
       const wrapper = shallow(<Panel channel={testChannel} api={testApi} />);
       const setKnobsHandler = handlers['addon:knobs:setKnobs'];
 
-      const knobs = {
-        foo: {
-          name: 'foo',
-          value: 'default string',
-          type: 'text',
-        },
-        baz: {
-          name: 'baz',
-          value: 'another knob value',
-          type: 'text',
-        },
-      };
+      const knobs = { foo: { name: 'foo', value: 'default string', type: 'text' }, baz: { name: 'baz', value: 'another knob value', type: 'text' } };
```

Comment:
@ndelangen related to my comment above, i'm surprised that the linter allows long lines like this?